components/blocks/FooterCallToAction.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <section class="w-full bg-primary-dark">
+    <div
+      class="base-wrapper flex flex-col lg:flex-row lg:justify-around lg:items-center py-14 lg:py-6"
+    >
+      <h2
+        class="text-white text-4xl leading-relaxed uppercase font-bold w-1/12 lg:w-1/2 pb-8 lg:p-0"
+      >
+        {{ block.text }}
+      </h2>
+
+      <div>
+        <BaseButton> {{ block.button[0].btnText }} </BaseButton>
+      </div>
```

Comment:
The base button needs an `:href` passed to it so it actually clicks through to the contact page

reference the button in the `HomepageHero`. should be a 1:1

Reply:
This button also needs to be `w-full` on mobile and `w-auto` from lg and up

Reply:
:href is added.

Added responsive button width classes and added "block" class to BaseButton component.

Thanks for all of your help.